db/seeds.rb

```diff
@@ -5,3 +5,10 @@
 #
 #   cities = City.create([{ :name => 'Chicago' }, { :name => 'Copenhagen' }])
 #   Mayor.create(:name => 'Emanuel', :city => cities.first)
+
+User.create!(
+  username: 'admin',
```

Comment:
I believe this default user could be specified in the README.md, so that the user wouldn't have to look into this file.